bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/TestScan.java

```diff
@@ -210,7 +210,7 @@ public void test100ResultsInScanner() throws IOException {
 
     Scan scan = new Scan();
     scan.withStartRow(rowKeys[0])
-        .withStopRow(rowFollowing(rowKeys[rowsToWrite - 1]))
+        .withStopRow(rowFollowing(rowKeys[rowsToWrite - 1]), true)
```

Comment:
I don't think this is right either, this changed the test behavior and made the stop key inclusive

Reply:
fixed